doc/tutorial/10_grafana.md

```diff
@@ -0,0 +1,36 @@
+# Grafana Dashboard
+
+This tutorial will show you how to encapsulate prometheus standard API and use them to monitor state of pgmoneta with the help of grafana dashboard.
+
+## Preface
+
+This tutorial assumes that you have an installation of PostgreSQL 10+ and pgmoneta.
+
+See [Install pgmoneta](https://github.com/pgmoneta/pgmoneta/blob/main/doc/tutorial/01_install.md)
+for more detail.
+
+## Prometheus Configuration
+
+First of all, you should install prometheus in your pgmoneta server. For `Ubuntu 22.04`, you can just use `apt-get` command to install it.
```

Comment:
prometheus -> Prometheus

Remove "For `Ubuntu 22.04`, you can just use `apt-get` command to install it."